### PR TITLE
MES-2750: Passing null for passCompletion

### PR DESCRIPTION
**Pull request by szabikr**
## Description

Setting up the correct data structure

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/modules/tests/tests.cat-adi-part2.reducer.ts [hunk 1]
```diff
@@ -34,7 +33,7 @@ export function testsCatADIPart2Reducer(
       accompaniment: accompanimentReducer,
       vehicleDetails: vehicleDetailsCatADIPart2Reducer,
       testData: testDataCatADI2Reducer,
-      passCompletion: passCompletionReducer,
+      passCompletion: () => null,
```
Comment (TomBillingtonUK) [inline src/modules/tests/tests.cat-adi-part2.reducer.ts]: Is this a temporary thing?

Comment (szabikr) [reply src/modules/tests/tests.cat-adi-part2.reducer.ts]: No not really. It has to be null for ADI pt2 tests

Comment (TomBillingtonUK) [reply src/modules/tests/tests.cat-adi-part2.reducer.ts]: As it's common it can't be removed. Can we use the nullReducer instead.

Comment (TomBillingtonUK) [reply src/modules/tests/tests.cat-adi-part2.reducer.ts]: src/shared/classes/null.reducer.ts

**Review by ammarhaiderbjss — APPROVED** [review]

File: src/modules/tests/tests.cat-adi-part2.reducer.ts (hunk 1 above)
Comment (szabikr) [reply src/modules/tests/tests.cat-adi-part2.reducer.ts]: using `nullReducer` now

**Review by TomBillingtonUK — APPROVED** [review]

**Pull request merged**